corehq/apps/api/resources/v0_4.py

```diff
@@ -120,6 +120,7 @@ def obj_get_list(self, bundle, domain, **kwargs):
 
         def wrapper(doc):
             if doc['doc_type'] in xform_doc_types:
+                doc.pop('user_id', None)  # needed when xform is stored in sql
                 return xform_doc_types[doc['doc_type']].wrap(doc)
```

Comment:
This wasn't added by me. Originally added here: https://github.com/dimagi/commcare-hq/commit/d3f1c857d5721e5081b44f1d63dde17db1f3baa7 when we started using ES as the API backend (perviously we didn't have a 'form list' API).

The reason for the wrapping is that Tastypie is designed to work off model instances. After looking at this a bit more I suspect the API's are broken in even more ways e.g. I doubt this works: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/api/resources/v0_4.py#L64-L69